frontend/src/components/Navbar.tsx

```diff
@@ -0,0 +1,41 @@
+import classNames from 'classnames'
+import React from 'react'
+import { BsWallet } from "react-icons/bs"
+import cn from 'classnames'
+
+function Navbar() {
+  return (
+    <div className='px-5 py-3 bg-slate-900 items-center shadow-lg mx-auto flex justify-between'>
+        <a href='/'><h1 className='text-white font-bold text-2xl'>ZILLIQA</h1></a>
+        <button className={
```

Comment:
Use the Button component instead

Reply:
I have corrected it

Reply:
You need to push your commits when you have made any new changes